cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -53,6 +53,7 @@ var (
 	_                    = deprecatedflags.Add("connection-timeout")
 	volumeNamePrefix     = flag.String("volume-name-prefix", "pvc", "Prefix to apply to the name of a created volume.")
 	volumeNameUUIDLength = flag.Int("volume-name-uuid-length", -1, "Truncates generated UUID of a created volume to this length. Defaults behavior is to NOT truncate.")
+	volumeNamesReadable  = flag.Bool("volume-names-readable", false, "If enabled, includes the PVC namespace and name in VolumeRequests' suggested names.  Note that, combined with --volume-name-uuid-length, this can cause naming collisions.")
```

Comment:
> this can cause naming collisions

Name collisions will lead to a PVC that will be Pending forever, because name of the PV it wants collided with another PVC. Users should not face these collisions at all, they're unpredictable and hard to fix - it requires users to change deployments of their applications.